Shell/Preprocess.cpp

```diff
@@ -107,6 +108,17 @@ void Preprocess::preprocess(Problem& prb)
     Normalisation().normalise(prb);
   }
 
+  if (_options.shuffleInput()) {
+    TimeCounter tc(TC_SHUFFLING);
+    env.statistics->phase=Statistics::SHUFFLING;    
+
+    if (env.options->showPreprocessing())
+      env.out() << "shuffling1" << std::endl;
+
+    // shuffling at the very beginning
```

Comment:
Ah, so we do have a shuffle here! So...why also in the various algorithms?

Reply:
Shuffling the input is what Geoff (allegedly) does to our inputs in the competition. 

But it's also interesting to study a fixed input and noise in the algorithms. 

Didn't have a careful look at the difference between the two yet, but the two options create some conceptual separation.